### ISSUE TITLE
emissary quest frame appears when it shouldn't

### ISSUE THREAD

**Issue by kived** [mikepauer/Carbonite#615]
After opening and closing the large map, the emissary quest frame appears in the middle of the screen. This happens only with the most recent commit (https://github.com/mikepauer/Carbonite/commit/0a84bd506fdc4ea3143e599893e46fd26585b689)

![screen shot 2017-02-26 at 12 45 33 pm](https://cloud.githubusercontent.com/assets/1126441/23343517/d1f4978c-fc21-11e6-9048-62b97cb017b7.png)

**Comment by IrcDirk** [comment]
Ok now it should be fixed. Please check and report back.

**Comment by kived** [comment]
Looks good! Thanks!

**Comment by kived** [comment]
So the fix has introduced another issue. I started seeing the "You will receive another emissary quest tomorrow" tooltip on top of the small map randomly (i.e. when my mouse was anywhere at all on the screen, no place in particular triggered it), and when it appears it interferes with clicking on things. This started happening when I was in our alt raid last night - I almost got hit by Focused Blast a few times since I use the mouse a lot for movement, and I had to resort to tab targeting *gasp*. :P
